assets/js/app.js

```diff
@@ -76,6 +93,89 @@ $(function () {
       }
   });
 
+  /*JavaScript for toggle for light/dark mode*/
+  
+    var checkbox = document.querySelector("#switch-1");
+    var toggler = document.querySelector('.toggle-button-1');
+  
+
+  if (checkbox) {
+    let a = localStorage.getItem("theme");
+    document.documentElement.setAttribute("data-theme", a); // setting the initial theme to light
+
+    if (localStorage.getItem("theme") === "dark") {
+      checkbox.checked = true;
+    }
+
+    // selecting the footer text & footer icons
+    const footerText = document.querySelector('.footer-text p');
+    const footerIcons = document.querySelectorAll('.sub-footer-icon');
```

Comment:
What is the use of footer Icon constant if they are not implemented in code anywhere